omniport/core/session_auth/views/logout.py

```diff
@@ -22,9 +26,11 @@ def get(self, request, *args, **kwargs):
         :return: the response for request
         """
 
+        user = request.user
+        
         # This is a direct replacement for django.contrib.auth.logout()
         SessionMap.delete_session_map(request=request)
-
+        session_auth_log('Successfully logged out', 'info', user)
```

Comment:
Log message contains no information: Who got successfully logged out and which session?

Reply:
Done: added session key, and user info